cmd/mattermost/commands/channel_test.go

```diff
@@ -131,3 +131,16 @@ func TestRenameChannel(t *testing.T) {
 	assert.Equal(t, "newchannelname10", updatedChannel.Name)
 	assert.Equal(t, "New Display Name", updatedChannel.DisplayName)
 }
+
+func Test_searchChannelCmdF(t *testing.T) {
+	th := Setup().InitBasic()
+	defer th.TearDown()
+
+	channel := th.CreatePublicChannel()
+
+	th.CheckCommand(t, "channel", "search", channel.Name)
+	th.CheckCommand(t, "channel", "search", channel.Name+"404")
```

Comment:
You are checking that the command runs, and doesn't fail, but you are not checking that the command does what is expected. Here is needed to test the output too.